.github/workflows/cd.yml

```diff
@@ -74,7 +73,7 @@ jobs:
           ${{ env.GCR_BASE }}/${{ env.GITHUB_REPOSITORY_SLUG_URL }}/${{ env.GITHUB_REPOSITORY_SLUG_URL }}/${{ env.GITHUB_HEAD_REF_SLUG || env.GITHUB_REF_SLUG }}:latest
           ${{ env.GCR_BASE }}/${{ env.GITHUB_REPOSITORY_SLUG_URL }}/${{ env.GITHUB_REPOSITORY_SLUG_URL }}/${{ env.GITHUB_HEAD_REF_SLUG || env.GITHUB_REF_SLUG }}:${{ env.GITHUB_SHA_SHORT }}
         build-args: |
-          NETWORK=${{ github.event.inputs.network || env.NETWORK }}
+          NETWORK=${{ github.event.inputs.network }}
```

Comment:
`Mainnet` inside the Dockerfile unless overridden by manual deploy, right?

Reply:
Correct